### PR TITLE
Move Email Activities to Network Category

### PR DESCRIPTION
**Pull request by ablyler**
Email-related activities better fit the network category rather than the system category; this also allows email activities not to require an actor attribute, which is hard to determine what the appropriate value should be.

### CODE REVIEW

**Comment by rroupski** [comment]
I don't think we need any email class(es) in the Findings category. Instead, we should use the email classes in the network category with a profile, e.g. Malware profile, for events that include detections of bad things found in the email.

**Review by rroupski — CHANGES_REQUESTED** [review]
See my comment

File: extensions/dev/events/findings/email/email.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "caption": "Email Security Finding",
+  "category": "findings",
   "name": "email_finding",
```
Comment (Aniak5) [inline extensions/dev/events/findings/email/email.json:4]: Agreed with @rroupski that we should remove `email` from the `findings` category.

**Comment by ablyler** [comment]
@rroupski + @Aniak5 thanks for the feedback, updates have been made.

File: extensions/dev/events/network/email/email.json [hunk 2]
```diff
@@ -26,4 +34,4 @@
       "requirement": "optional"
     }
```
[the remark this answers is not in the document]
Comment (ablyler) [reply extensions/dev/events/network/email/email.json:35]: I am okay with removing it, it seems like something we could add back if we need to link email objects with the vendor's quarantine system.

**Comment by Aniak5** [comment]
@ablyler did you mean to move the `peripheral_activity` class from the `system` to the `network` category?

**Comment by ablyler** [comment]
> @ablyler did you mean to move the `peripheral_activity` class from the `system` to the `network` category?

Good catch, that was not intentional. I have fixed that now.

**Review by Aniak5 — APPROVED** [review]
LGTM!

**Review by rroupski — APPROVED** [review]

**Pull request merged**